plugins/modules/helm.py

```diff
@@ -573,7 +574,7 @@ def main():
         else:
 
             if has_plugin(helm_cmd_common, "diff") and not chart_repo_url:
-                would_change = helmdiff_check(module, helm_cmd_common, release_name, chart_ref,
+                would_change, diff = helmdiff_check(module, helm_cmd_common, release_name, chart_ref,
                                               release_values, values_files, chart_version, replace)
```

Comment:
The indentation here needs to be corrected in order to address the linting error.